consumers/aptos.md

```diff
@@ -4,33 +4,6 @@ description: Consume Pyth Network prices in applications on Aptos
 
 # Pyth on Aptos
 
-## How Pyth on Aptos Works
-
```

Comment:
I've copied this shared text and am going to move it to the "how pyth works" section in the next PR. I've put a summary of the relevant parts for developers in `consume-data.md`

Reply:
100% this is the right thing to do.